Source/WTF/wtf/StringPrintStream.cpp

```diff
@@ -86,7 +86,7 @@ void StringPrintStream::vprintf(const char* format, va_list argList)
 CString StringPrintStream::toCString()
 {
     ASSERT(m_next == strlen(m_buffer));
-    return CString(m_buffer, m_next);
+    return CString({ m_buffer, m_next });
```

Comment:
Can we omit `CString()` here?

Reply:
Maybe not, I don’t see how it figures out to use `std::span`.

Reply:
If I remember correctly, it was either:
```
return CString({ m_buffer, m_next });
```
or
```
return std::span<const char> { m_buffer, m_next };
```
(it guess because m_buffer is not `const`.

I went with the most concise version that built.